fontbe/src/paths.rs

```diff
@@ -76,7 +81,9 @@ impl Paths {
             WorkId::Hhea => self.build_dir.join("hhea.table"),
             WorkId::Hmtx => self.build_dir.join("hmtx.table"),
             WorkId::Hvar => self.build_dir.join("hvar.table"),
-            WorkId::Kerning => self.build_dir.join("kerning.bin"),
+            WorkId::KernPairs => self.build_dir.join("kern_scatter.bin"),
```

Comment:
KernPairs=>"kern_scatter"? why not make the names of the structs and paths (and timing labels) be consistent with one another?